core/db/migrate/20221215151408_add_selected_locale_to_spree_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSelectedLocaleToSpreeUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :spree_users, :selected_locale, :string
```

Comment:
```suggestion
    add_column Spree.user_class.table_name.to_s, :selected_locale, :string, if_not_exists: true
```

1. This will always use the correct table name
2. Will not create this column if an app developer created it independently earlier